src/privatesend.h

```diff
@@ -409,8 +397,8 @@ class CPrivateSend
 
     /// If the collateral is valid given by a client
     static bool IsCollateralValid(const CTransaction& txCollateral);
-    static CAmount GetCollateralAmount() { return COLLATERAL; }
-    static CAmount GetMaxCollateralAmount() { return COLLATERAL*4; }
+    static CAmount GetCollateralAmount() { return vecStandardDenominations.back()/10; }
```

Comment:
Would prefer to not assume knowledge of `vecStandardDenominations` here and instead use some kind of `GetSmallestDenomination`, but I guess we can do some clean up later.

(Only because this exposes internals of how we manage PS denoms, and we are directly accessing the internal vector instead of using accessors, which provide an abstraction and allow calling code to not worry about implementation.)